operations/mimir-mixin/dashboards/dashboard-queries.libsonnet

```diff
@@ -25,55 +27,104 @@
     query_http_routes_regex: '(prometheus|api_prom)_api_v1_query(_range)?',
 
     gateway: {
+      // deprecated, will be removed
       writeRequestsPerSecond: 'cortex_request_duration_seconds_count{%(gatewayMatcher)s, route=~"%(writeHTTPRoutesRegex)s"}' % variables,
       readRequestsPerSecond: 'cortex_request_duration_seconds_count{%(gatewayMatcher)s, route=~"%(readHTTPRoutesRegex)s"}' % variables,
 
+      local p = self,
+      writeRequestsPerSecondMetric: 'cortex_request_duration_seconds',
```

Comment:
Are there cases where `writeRequestsPerSecondMetric` and `readRequestsPerSecondMetric` would point to different metrics? If not, why not merge them into one `requestsPerSecondMetric`?

Reply:
Not sure where that comes from, I may as well try and we'll see if it causes some problem down the line, thanks.